src/utils/schema/resolvers/Query/all.ts

```diff
@@ -533,6 +534,45 @@ export default (entityName: string, fieldsByName: any, idsByName: any) =>
         }
         return arr;
       }, []);
+
+      const promises = fields.map(async (field) => {
```

Comment:
@morgang-adappt @HasnatL-Adapptlimited 

I don't know who I should tag as I can see many people worked on these branches

This is breaking some code, so either I need to comment it and someone should fix it later, or, we correct the issues ( it's not the only one from what I noticed ) before we merge & deploy

for example, if I have a question like that:

![Capture d’écran 2024-10-21 à 13 07 34](https://github.com/user-attachments/assets/841978c1-b7f2-4903-aae3-f6aaab18d851)

Now the data in grids is queried like that:

![Capture d’écran 2024-10-21 à 13 08 50](https://github.com/user-attachments/assets/69faf2f8-2c30-4b5a-8dfa-b6fb91f38f16)

and if I try to do some inline edition, it's not working anymore:

![Capture d’écran 2024-10-21 à 13 08 55](https://github.com/user-attachments/assets/c04ea885-e3a9-4332-8755-b9ac9832d9bc)


I'll continue to check but I'm pretty sure other things are broken due to this change

Reply:
Hi @AntoineRelief,

I've been discussing the issue with Vaishali and the team. We can see that the option keys are no longer part of the query response, but are having a hard time recreating the issue with inline edition. Do you have any availability to go on a call and talk through the issue?

Reply:
@morgang-adappt 

sure, it really just take a minute to reproduce
I can be available from 3 CEST to 5 CEST today so feel free to send me an invitation, or, if you want I can record a video to explain the issue and how to reproduce it

Reply:
Hi Antoine,

A video would be greatly appreciated! Thank you